### ISSUE TITLE
[Visual Studio Code Recent Projects] Enter opens finder and not VS Code

### ISSUE THREAD

**Issue by sriharshAmur** [raycast/extensions#6001]
### Extension

https://www.raycast.com/thomas/visual-studio-code

### Description

I installed the VS Code Recent Projects Extension. It shows me all my recent projects, however upon entering it opens the finder and not the VS Code application with that project. Did anybody else have the same issue and have a fix?

### Steps To Reproduce

1. Install the extension
2. open the extension in Raycast
3. Click on one of your recent files

### Current Behaviour

Opens the folder of the repository in Finder

### Expected Behaviour

Opens the folder in VS Code

**Comment by pernielsentikaer** [comment]
Hi @sriharshAmur 👋 

Are you using the right action here?

<img width="487" alt="Raycast-Q7JvZKI6@2x" src="https://user-images.githubusercontent.com/38033107/232679765-291e7273-3990-48c9-835e-b576ef182146.png">

**Comment by sriharshAmur** [comment]
@pernielsentikaer Yes, I see the same screen as you in the actions. However, When I press enter, it opens the files in Finder. FYI, I do have multiple VS Code windows open, but I doubt that should be the problem.

**Comment by pernielsentikaer** [comment]
Do you maybe have a wrong version deg

**Comment by sriharshAmur** [comment]
I had VS Code configured. However, it was not in my Applications folder. It was installed on my Desktop. After moving it to the Applications folder, it worked.

**Comment by theherk** [comment]
This is the case for me. I am using VSCodium, and it is in the `/Applcations/` directory, and VSCodium is selected as the build in extension configuration.

**Comment by theherk** [comment]
I believe the reason is because the [`appKeyMapping`](https://github.com/raycast/extensions/blob/main/extensions/visual-studio-code-project-manager/src/search-project-manager-projects.tsx#L23) has `VSCodium: "com.visualstudio.code.oss",`.

In my case:

    jq .darwinBundleIdentifier /Applications/VSCodium.app/Contents/Resources/app/product.json
    "com.vscodium"

I think there are others for Codium, but this has caused other issues recently; see [here](https://github.com/VSCodeVim/Vim/issues/8052#issuecomment-1422383241) for example.

**Comment by theherk** [comment]
I have verified that changing this line to:

```tsx
  VSCodium: "com.vscodium",
```

Fixes the issue, but I'm hesitant to submit it yet, because I'm unsure why and in what conditions there are varying build identifiers. Will look into that.

**Comment by pernielsentikaer** [comment]
Closed as fixed
